src/SourceBuild/tarball/content/repos/xliff-tasks.proj

```diff
@@ -17,6 +17,8 @@
     <DeterministicBuildOptOut>true</DeterministicBuildOptOut>
     <IsToolingProject>true</IsToolingProject>
 
+    <ShippingPackagesOutput>$(ProjectDirectory)artifacts/packages/$(Configuration)/Shipping/</ShippingPackagesOutput>
```

Comment:
With ArPow, the intermediate NuGet packages get placed in nonshipping.  This is already the default therefore I'm not certain this is necessary.  All I think that should be necessary is to set `<OutputPlacementRepoApiImplemented>false</OutputPlacementRepoApiImplemented>`.  FYI, There is a follow up issue to cleanup the RepoApi.